### PR TITLE
Add timezone string after the event datetime section.

### PR DESCRIPTION
**Pull request by ghost**
It does what is stated in the title.

### CODE REVIEW

File: templates/jv_add_timezone_string_after_datetime.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?php
+
+// Remove the <?php tag if added to your functions.php
+// It allow you to add your timezone (i.e GMT+3) after the date time event information.
+// Replace the "your timezone string" by whatever you want.
+add_filter( 'FHEE__espresso_list_of_event_dates__datetime_html', 'jv_add_timezone_string_after_datetime', 10, 2 );
```
Comment (Pebblo) [inline templates/jv_add_timezone_string_after_datetime.php]:
```suggestion
add_filter( 'FHEE__espresso_list_of_event_dates__datetime_html', 'jv_ee_add_timezone_string_after_datetime', 10, 2 );
```

File: templates/jv_add_timezone_string_after_datetime.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+function jv_add_timezone_string_after_datetime( $datetime_html, EE_Datetime $datetime ) {
```
Comment (Pebblo) [inline templates/jv_add_timezone_string_after_datetime.php]:
```suggestion
function jv_ee_add_timezone_string_after_datetime( $datetime_html, EE_Datetime $datetime ) {
```

File: templates/jv_add_timezone_string_after_datetime.php [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+	$date_format = get_option( 'date_format' );
+	$time_format = get_option( 'time_format' );
+	return '
+	<span class="dashicons dashicons-calendar"></span>
+	<span class="ee-event-datetimes-li-daterange">' . $datetime->date_range($date_format) . '</span>
+	<br/>
+	<span class="dashicons dashicons-clock"></span>
+	<span class="ee-event-datetimes-li-timerange">' . $datetime->time_range($time_format) . ' your timezone string</span>
+	';
+}
```
Comment (Pebblo) [inline templates/jv_add_timezone_string_after_datetime.php]:
```suggestion
}

```

**Review by Pebblo — CHANGES_REQUESTED** [review]
Function name not using our naming convention;

Filename could also do with being changed to `jv_ee_add_timezone_string_after_datetime.php`;

**Comment by ghost** [comment]
@Pebblo I have applied the changes to follow the standards:
https://github.com/eventespresso/ee-code-snippet-library/pull/18/commits/7016b4209ea823e10a3c7d3d116997be2142d74d

**Pull request merged**